sh.cider.Cider.metainfo.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <content_rating type="oars-1.0" />
+  <id>sh.cider.Cider</id>
+  
+  <name>Cider</name>
+  <summary>An electron based Apple Music front-end</summary>
+  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>AGPL-1.0-or-later</project_license>
+
+  <url type="homepage">https://cider.sh/</url>
+  <url type="bugtracker">https://github.com/ciderapp/Cider/issues</url>
+  <url type="donation">https://ko-fi.com/cryptofyre</url>
+  
+  <description>
+    <p>
+      Project Cider. A new look into listening and enjoying Apple Music in style and performance.
```

Comment:
Could it be expanded to follow marketing blurb from https://cider.sh/?

Reply:
I used the description from the Git repo for that field what would you like to see me replace it with from their site? There's not much other than the features and the summary you suggested.